koku/providers/ibm/provider.py

```diff
@@ -0,0 +1,46 @@
+"""IBM provider implementation to be used by Koku."""
```

Comment:
Can you make this be the standard Copyright message that's used in the other python files?